tracing-core/src/field.rs

```diff
@@ -219,6 +219,8 @@ pub trait Value: crate::sealed::Sealed {
 }
 
 /// A `Value` which serializes as a string using `fmt::Display`.
```

Comment:
Perhaps we should also remove the "as a string", as well, if this was a source of confusion?